package.json

```
@@ -51,8 +51,8 @@
     "use-material-ui-next": "node scripts/useMaterialUINext.mjs",
     "pkg-pr-new:install": "pnpm install --no-frozen-lockfile",
     "pkg-pr-new:build": "pnpm run release:build",
-    "pkg-pr-new:packages": "find packages -type d -name 'x-*' -exec test -d {}/build \\; -print | tr '\n' '#' | sed 's|#|/build |g'",
-    "pkg-pr-new:release": "pnpm dlx pkg-pr-new publish $(pnpm -s pkg-pr-new:packages) --template './examples/*' --comment=off --peerDeps",
+    "pkg-pr-new:packages": "tsx ./scripts/printPkgPrNewPublishDirs.mts",
+    "pkg-pr-new:release": "pnpm dlx pkg-pr-new publish $(pnpm -s pkg-pr-new:packages) --pnpm --packageManager=pnpm --template './examples/*' --comment=off --peerDeps",
```

Try reverting `pnpm pack` usage
```suggestion
    "pkg-pr-new:release": "pnpm dlx pkg-pr-new publish $(pnpm -s pkg-pr-new:packages) --packageManager=pnpm --template './examples/*' --comment=off --peerDeps",
```